### PR TITLE
Ash drake loot: Drakelings

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# THEOS

## I had to grind for 2 years to get mine

Adds a 4th loot item to the ash drake: a dragon egg
throw the dragon egg in lava (this is perfectly safe trust me) to incubate it and hatch your very own baby dragon!
drakelings can be renamed with a collar (doesn't actually use the collar) and used as a mount that flies so it can cross lava and stuff
Drakelings also have two attacks, which can be used by either their rider or the drakeling itself if it's sentient:
Fire Breath: similar to an ash drake's fire breath but much shorter, deals 20 damage on lavaland with 4 range (bonus of 60 vs lavaland mobs) and 5 damage on station with 1 range, 2 second cooldown
Wing Flap: shoots 3 projectiles in front of the drakeling that deal 40 damage to lavaland mobs and push things away slightly, each projectile can also break rocks with the same effectiveness as an advanced plasmacutter, 2 second cooldown

you can also do an ollie on your dragon which is a cool way to style on the other miners. You can also grind similarly to a skateboard but still have directional control.

the drakeling can be healed similarly to other mobs but can also be healed by feeding it goliath meat, with cooked meat being more effective than uncooked

# THEOS
# WHY
### Are you adding more mining loot
because it's cool and isn't something that just kills people ok
it's actually able to like
mine good instead of stabbing things to death

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.
The ash drake's loot table now contains a dragon egg which can be hatched by incubating it with lava
the drakeling that hatches can be named with a pet collar and will help you fight stuff
it can also be used as a mount to fly around over lava and chasms without having wings yourself
drakelings have two attacks: a fire  breath attack that deals high damage to lavaland fauna, and a wing flap attack that can shred rocks like a plasmacutter
the drakeling can be healed similarly to other mobs but can also be healed by feeding it goliath meat, with cooked meat being more effective than uncooked
### What should players be aware of when it comes to the changes your PR is implementing?
DRAGON
PET
### What general grouping does this PR fall under? 
mining

# Changelog

:cl:  
rscadd: adds a new item to the ash drake's loot table, a dragon egg that can be hatched into your very own pet dragon to help you mine and shoot things! It's not very good at killing people but it tries its best.
imageadd: drakeling ollie image and dead drakeling icon
/:cl:

### CODE REVIEW

**Comment by LazennG** [comment]
DEMENTED coder TERRORIZES server by providing miners with HUGE FIRE LIZARD

**Comment by ChesterTheCheesy** [comment]
Finally a tool that allows us to reliably break into science to get our bags of holdings.

**Comment by ToasterBiome** [comment]
We MMO now boyssss

Add little bubblegum too

**Comment by Yogbot-13** [comment]
#### Test Merge Deployed

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: 03380ff77741c1f53110a9efea8d92845f68f041
Server: 576b980d7860d3a2d8af75956984af14a64580ef

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: 03380ff77741c1f53110a9efea8d92845f68f041
Server: ce13dee3d6e20b89dc7daedfb859ee58550b7dcf

**Comment by LazennG** [comment]
frick drake walks into airlocks before the rider so i gotta throw my id at doors

**Comment by SomeguyManperson** [comment]
just click the door to open it?

**Review by Partheo — APPROVED** [review]
dragon content
